style.css

```diff
@@ -1 +1,263 @@
+* {
+ box-sizing: border-box;
+}
+.body {
+ background-color: #e6e2e1;
+}
 
+/*Nav Bar*/
+
+.nav-flex {
+ display: flex;
+ flex-direction: row;
+ justify-content: flex-end;
+ width: 100%;
+ height: 200px;
+ vertical-align: middle;
+}
+.sign {
+ background-color: #3665d9;
+ font-size: 15px;
+ font-family: Verdana, Geneva, Tahoma, sans-serif;
+ color: white;
+ margin: 4px;
+ border-radius: 5px;
+ border: 1px solid black;
+ width: 110px;
+ height: 35px;
+}
+#block {
+ margin-right: 6px;
+}
+.nav-align p a {
+ color: #33ace8;
+ text-decoration: none;
+}
+.nav-align p {
+ padding: 0px;
+ margin: 2px;
+}
+
+/* form section */
+
+.main-col {
+ width: 100%;
+ height: auto;
+}
+.flex-container {
+ display: flex;
+ flex-direction: column;
+ align-items: center;
+ justify-content: center;
+ margin-top: -100px;
+}
+.flex-sub-logo {
+ display: flex;
+ flex-direction: row;
+ flex-flow: row;
```

Comment:
This property is not being properly used, take a read [here](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-flow).
```suggestion
```